lms/djangoapps/discussion_api/api.py

```diff
@@ -123,3 +127,78 @@ def get_thread_list(request, course_key, page, page_size):
 
     results = [ThreadSerializer(thread, context=context).data for thread in threads]
     return get_paginated_data(request, results, page, num_pages)
+
+
+def get_comment_list(request, thread_id, endorsed, page, page_size):
+    """
```

Comment:
The full Markdown-based format seems like overkill to me in cases where we are not planning to actually extract a docstring and include it in the published documentation. Thoughts?